src/components/_button.scss

```diff
@@ -8,10 +8,10 @@
   $type: 'primary',
   $size: 'medium',
   $layout: null,
-  $color: null,
-  $bgcolor: null,
-  $icon-fill: null,
-  $icon-stroke: null
+  $background-color: $c-gray-dark,
+  $color: $c-white,
+  $icon-fill: transparent,
+  $icon-stroke: $c-white
```

Comment:
it was just to add context to the variable, but the PR is fine like this too. No prob.